monero-outreach-docs/translations/pt-br/quickfacts/QuickFacts_pt-br.md

```diff
@@ -0,0 +1,125 @@
+# MONERO
+
+Monero é uma moeda digital segura, privada, não-rastreável que está disponível para todos. As suas caracterísitcas o tornam uma moeda fungível.
+
+# FATOS RÁPIDOS
+
+Usuários podem gastar com segurança, ninguém consegue ver seu saldo ou rastrear suas atividades sem sua permissão.
+
+Revisado em 1/2/2019 - [Printer Friendly Version](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts_PrinterFriendly.pdf)
+
+- [Versão colorida](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts_pt.pdf)
+
+[getmonero.org](https://getmonero.org/)
+
+
+## Princípios Fundamentais
+
+#### DESCENTRALIZAÇÃO
+
+Sendo um _[projeto código aberto](https://github.com/monero-project/monero)_ liderado e _[financiado](https://forum.getmonero.org/8/funding-required)_ por um _[time decentralized](https://getmonero.org/community/team/)_ de desenvolvedores e membros da comunidade, o mesmo não pode ser censurado. A maioria dos contribuidores são voluntários e a comunidade está espalhada por todo o globo.
+
+#### SEGURANÇA
+
+Sendo uma criptomoeda descentralizada, o Monero é seguro por uma grande rede de usuários espalhados pelo mundo. Transações são confirmadas por um consenso distrubuído e depois gravadas imutavelmente no _[blockchain](https://www.mycryptopedia.com/what-is-blockchain-technology/)_.
+
+#### ESCALABILIDADE
+
+Diferente do Bitcoin e de outras moedas, as quais costumam ter constantes de código inalterável (_hardcoded constants_), o _[tamanho do bloco](https://www.mycryptopedia.com/block-size-explained/)_ do Monero é adapatado automaticamente ao volume de transações, grantindo menores taxas e confirmações mais rápidas.
+
+#### PRIVACIDADE
+
+Monero usa criptografia sofisticada como _[assinaturas em anel](https://getmonero.org/resources/moneropedia/ringsignatures.html)_, _[transações confidenciais em anel](https://www.mycryptopedia.com/monero-ring-confidential-transactions-ringct/)_, e _[endereços stealth](https://www.mycryptopedia.com/everything-need-know-stealth-addresses/)_ para ofuscar as origens, quantidades e destinatários de todas as transações. Os usuários podem decidir exatamente quais informações e para quem eles querem revelar.

```

Comment:
- _assinaturas em anel_: https://www.getmonero.org/pt-br/resources/moneropedia/ringsignatures.html
- _transações confidenciais em anel_: https://www.getmonero.org/pt-br/resources/moneropedia/ringCT.html
- _endereços stealth_/**Endereços Sigilosos**: https://www.getmonero.org/pt-br/resources/moneropedia/stealthaddress.html

In getmonero.org you can find the translation for "stealth address" as **Endereços Sigilosos**, is good to verify if this is used among translators. Just leaving it for the reviewer also :).

Reply:
@lh1008 Regarding "stealth address" translation, I would argue that both are correct. [Here](https://github.com/monero-ecosystem/monero-translations/blob/master/terminology-guides/portuguese-br-terminology.md) you can find where @botoserelepe probably found this terminology.

Reply:
Ok remember we have to try and have consistency in the terminologies around the community so there won't be much confusion about the meanings. That is why I pointed out.